omnij-jsonrpc/src/main/java/foundation/omni/rpc/OmniClient.java

```diff
@@ -2,24 +2,17 @@
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.JavaType;
-import com.google.common.primitives.UnsignedBytes;
-import org.consensusj.bitcoin.rpc.BitcoinExtendedClient;
-import foundation.omni.json.pojo.OmniPropertyInfo;
-import org.bitcoinj.core.LegacyAddress;
-import org.bitcoinj.core.SegwitAddress;
-import org.consensusj.bitcoin.rx.jsonrpc.RxBitcoinClient;
-import org.consensusj.jsonrpc.JsonRpcException;
-import org.consensusj.bitcoin.rpc.RpcConfig;
-import foundation.omni.CurrencyID;
-import foundation.omni.Ecosystem;
-import foundation.omni.OmniValue;
-import foundation.omni.PropertyType;
+import foundation.omni.*;
 import foundation.omni.json.conversion.OmniClientModule;
+import foundation.omni.json.pojo.OmniPropertyInfo;
 import foundation.omni.net.OmniNetworkParameters;
 import org.bitcoinj.core.Address;
 import org.bitcoinj.core.Coin;
 import org.bitcoinj.core.NetworkParameters;
 import org.bitcoinj.core.Sha256Hash;
+import org.consensusj.bitcoin.rpc.RpcConfig;
+import org.consensusj.bitcoin.rx.jsonrpc.RxBitcoinClient;
+import org.consensusj.jsonrpc.JsonRpcException;
```

Comment:
All these changes to import statements are distracting from the actual change that was made. I'm sure this was done by IntelliJ. It also looks like I should probably make a global "Organize Imports" refactoring and check that in soon.

Reply:
Right, it was IntellJ.